mobile/src/reducers/auth/auth.epics.ts

```diff
@@ -51,18 +52,28 @@ export const startLoginProcessEpic$ = (action$: ActionsObservable<StartLoginProc
     );
 
 //----------------------------------------------------------------------------
-export const startLogoutProcessEpic$ = (action$: ActionsObservable<StartLogoutProcess>, _: StateObservable<AppState>, dep: DependenciesContainer) =>
+function logout(dependencies: DependenciesContainer, installId?: string) {
+    if (installId) {
+        notificationsUnregister(dependencies, installId).catch(console.warn);
```

Comment:
As far as I can see the notificationsUnregister contains only the appropriate server call.

If this server call won't succeed, the app will still get push notifications in logout state and try to open profile in logout state, am I right? 
 
If yes is there an app center api to prevent getting notifications in logged out state?

If there is no such api, an additional check in onPushNotificationReceived lister is required IMHO.

Reply:
You are absolutely right, but, here is the problem - we cannot, or do not know, how to process silent Push Notifications. In all other cases notification will appear before we could deal with it. So, we sure can check whether user is logged in or compare employeeId but till we do not receive notifications silently it still will be a palliative.

Reply:
Ok, but are you sure you won't get a crash or an unusual behaviour in case of getting notifications on welcome screen and trying to open profile?

Reply:
Makes sense again. We have to check whether app is authorised and compare employeeId

Reply:
0bf229dbb8bc6bd65937977fdb221da97c6036dc